Using updated TPM submodule (from the second PR), I am hitting the following on make os:

  CC      /home/jirka/repos/lsdk/build/fTPM/./lib/tpm/tpm_symlink/TPMCmd/tpm/src/command/Asymmetric/ECC_Parameters.o
In file included from ./lib/tpm/tpm_symlink/TPMCmd/tpm/include/Global.h:79:0,
                 from ./lib/tpm/tpm_symlink/TPMCmd/tpm/include/Tpm.h:46,
                 from lib/tpm/tpm_symlink/TPMCmd/tpm/src/command/Asymmetric/ECC_Parameters.c:35:
./lib/tpm/tpm_symlink/TPMCmd/tpm/include/TpmTypes.h:1643:36: error: ‘RSA_PRIVATE_SIZE’ undeclared here (not in a function); did you mean ‘TPM_PARAMETER_SIZE’?
         BYTE                buffer[RSA_PRIVATE_SIZE];
                                    ^~~~~~~~~~~~~~~~
                                    TPM_PARAMETER_SIZE

Will update the module to use the older version, but it seems that we will hit that on the current master as well.

--- a/Makefile
+++ b/Makefile
@@ -27,8 +27,8 @@ endif
 all: firmware os
 
 .PHONY: firmware os
-firmware: u-boot-signed ppa-optee bootscript
-os: linux rfs
+firmware: u-boot-signed ppa-optee
+os: linux rfs bootscript
 
 .PHONY: u-boot-signed
 u-boot-signed: u-boot $(O)/hdr_spl.out

--- a/Readme.md
+++ b/Readme.md
@@ -81,8 +81,6 @@ This will build the following items:
    * `build/hdr_spl.out`
  * OP-TEE OS and PPA
    * `build/ppa.itb`
- * Boot script
-   * `build/grapeboard_boot.scr`
 
 These files must be written to NOR flash.
 

